cdc/processor/sinkmanager/table_sink_wrapper.go

```diff
@@ -89,13 +91,14 @@ func newTableSinkWrapper(
 	}
 }
 
-func (t *tableSinkWrapper) start(replicateTs model.Ts) {
+func (t *tableSinkWrapper) start(startTs model.Ts, replicateTs model.Ts) {
 	log.Info("Sink is started",
 		zap.String("namespace", t.changefeed.Namespace),
 		zap.String("changefeed", t.changefeed.ID),
 		zap.Int64("tableID", t.tableID),
 		zap.Uint64("replicateTs", replicateTs),
 	)
+	t.checkpointTs.Store(startTs)
```

Comment:
What about using `t.startTs` here?

Reply:
It may have been synced on another node at another time node. So the startTs here are not the same as the original ones.